ui/page/creatorMemberships/supporterArea/index.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import MembershipsPage from './view';
```

Comment:
:small_red_triangle: Tier checkbox not responsive if created out of order

- Create 6 tiers.
- Delete the last 2.
- Click "Add a Tier" twice to open the last 2 slots.
    - The last slot's checkbox is not responsive, and sometimes checks the one in slot 5

<img width="558" alt="image" src="https://user-images.githubusercontent.com/64950861/191071687-f2ab5cbb-d904-4cd4-bff6-3cbb60c0bcf9.png">

Reply:
:small_red_triangle: Playing around with the 5th and 6th slot's creation, I've seen this out-of-sync scenario twice (1 missing)
I think having the same name might be the trigger

<img width="1270" alt="image" src="https://user-images.githubusercontent.com/64950861/191072413-e4759acc-8cbf-4cc5-b1f8-e4716b7a68ff.png">

Reply:
Should be fixed by 099621bf19aa017f4936b8ad48851cffb868bbe0